index.html

```diff
@@ -11,5 +11,8 @@
     <body>
         <div id="root"></div>
         <script type="module" src="/src/main.tsx"></script>
+        <script>
+            const global = globalThis
```

Comment:
I'm guessing you're not using this, if so delete it.

Reply:
I'm using a module eli 7echtou bel reference heki , i can't delete it .
does it harm other parts?